lib/github_changelog_generator/generator/generator_generation.rb

```diff
@@ -143,13 +143,27 @@ def get_string_for_issue(issue)
       encapsulated_title = encapsulate_string issue["title"]
 
       title_with_number = "#{encapsulated_title} [\\##{issue['number']}](#{issue['html_url']})"
+      unless @options[:issue_line_labels].empty?
+        title_with_number = issue_line_with_labels(title_with_number, issue)
+      end
       issue_line_with_user(title_with_number, issue)
     end
 
     private
 
+    def issue_line_with_labels(line, issue)
+      labels = if options[:issue_line_labels] == ["ALL"]
+                 issue["labels"]
+               else
+                 issue["labels"].select { |label| options[:issue_line_labels].include? label["name"] }
+               end
+      labels_string = labels.map { |label| " \[[#{label['name']}](#{label['url'].sub! 'api.github.com/repos', 'github.com'})\]" }.join("")
+      line = "#{line}#{labels_string}"
+      line
```

Comment:
I would be more comfortable with a function that only calculated the labels string from the `issue` data:

```ruby
def line_labels_for(issue)
  labels = if options[:issue_line_labels] == ["ALL"]
             issue["labels"]
           else
             issue["labels"].select { |label| options[:issue_line_labels].include?(label["name"]) }
           end
  labels.map { |label| " \[[#{label['name']}](#{label['url'].sub('api.github.com/repos', 'github.com')})\]" }.join("")
end
```

This could be used like

```ruby
if options[:issue_line_labels].present?
  title_with_number = "#{title_with_number}{line_labels_for(issue)}"
end
```